keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/telemetry/TelemetryEvent.kt

```diff
@@ -34,13 +34,6 @@ data class ResourceCheckSkipped(
   )
 }
 
-data class LockAttempt(
-  val success: Boolean
-) : TelemetryEvent()
-
-val LockAttemptSucceeded = LockAttempt(true)
-val LockAttemptFailed = LockAttempt(false)
-
```

Comment:
We were no longer using these. Unrelated to this PR but thought I might as well tidy it up